### ISSUE TITLE
Bad error when parsing with preset prefix

### ISSUE THREAD

**Issue by Womblue** [openflighthpc/flight-hunter#38]
If you attempt to parse a node in the buffer with a preset prefix without specifying a start index, an unhandled error is raised. This issue can be replicated by running

```
bin/hunter hunt
```
on the server, then
```
bin/hunter send --prefix node
```
on the client. Then on the server when attempting to parse without using `--start`:
```
bin/hunter parse
```
When the node is selected it immediately raises
```
hunter: undefined method `length' for nil:NilClass
```

**Comment by VoxSecundus** [comment]
My suggestion for what should happen here is for the parser to operate without trying to stick a suffix onto the end of the prefix. Could whoever picks up this issue please also include a bugfix in their branch to handle when `--start` is given a non-numeric value?

**Comment by VoxSecundus** [comment]
Resolved in #42
